app/src/main/res/layout/spotlight_fragment_test_activity.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <FrameLayout
+      android:id="@+id/test_spotlight_overlay_placeholder"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent" />
+
+    <TextView
+      android:id="@+id/sample_spotlight_target"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="This is a sample text view"
+      style="@style/TextViewStart"
+      android:layout_marginTop="80dp"
+      android:layout_marginStart="80dp"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent" />
+  </androidx.constraintlayout.widget.ConstraintLayout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
done